test/standard/formatter_test.rb

```diff
@@ -1,7 +1,7 @@
 require "test_helper"
 
 class Standard::FormatterTest < UnitTest
-  Offense = Struct.new(:corrected?, :line, :real_column, :message)
+  Offense = Struct.new(:corrected?, :line, :real_column, :message, :cop_name)
```

Comment:
I was really sad to have to change (almost) all the tests in the suite, but I actually don't know if it would be possible to implement this fix without doing so (though I can't really explain the reasoning behind this opinion).